test/imports-test.ts

```diff
@@ -388,11 +388,55 @@ var _moduleName = require('moduleName');
       `"use strict";
       var _mymodule = require('my-module');
       
-      (0, _mymodule.bar)();
+      _mymodule.bar.call(void 0, );
     `,
     );
   });
 
+  it("never breaks weird call expressions", () => {
+    assertResult(
+      `
+      import a from 'a';
+      import b from 'b';
+      (a)();
+      ((a))();
+      a(b)();
+      (a + b)();
+      new (a)();
```

Comment:
Technically this doesn't exercise your case, since it's surrounded by parens here, but no big deal.